### PR TITLE
Add PROTO Field Comment Test

### PR DESCRIPTION
**Pull request by DavidMansolino**
This PR adds a test that checks that all the PROTO field comment ends with a ".".

### CODE REVIEW

**Comment by DavidMansolino** [comment]
This will prevent such mistakes in the future: https://github.com/cyberbotics/webots/pull/1283#discussion_r368501551

:+1:

**Comment by stefaniapedrazzi** [comment]
:eyes:

File: projects/objects/road/protos/Road.proto [hunk 1]
```diff
@@ -21,25 +21,25 @@ PROTO Road [
...
   field        MFNode{RoadLine {}}  lines                     [ RoadLine { } ]        # Defines the property of each line separating two lanes.
   field        SFFloat              roadBorderHeight          0.15                    # Defines the height of the sidewalk.
-  field        MFFloat              roadBorderWidth           [ 0.8 ]                 # Defines the width of the sidewalk associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
-  field        SFBool               road                      TRUE                    # Defines the width of the sidewalk associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
+  field        MFFloat              roadBorderWidth           [ 0.8 ]                 # Defines the width of the sidewalk. associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
```
Comment (stefaniapedrazzi) [inline projects/objects/road/protos/Road.proto]: Why did you add a '.' in the middle of the sentence?

It seems that it was already correct before:
```suggestion
  field        MFFloat              roadBorderWidth           [ 0.8 ]                 # Defines the width of the sidewalk associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
```

File: projects/objects/road/protos/Road.proto [hunk 2]
```diff
@@ -21,25 +21,25 @@ PROTO Road [
...
+  field        SFBool               road                      TRUE                    # Defines the width of the sidewalk. associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
```
Comment (stefaniapedrazzi) [inline projects/objects/road/protos/Road.proto]: Why did you add a '.' in the middle of the sentence?

It seems that it was already correct before:
```suggestion
  field        SFBool               road                      TRUE                    # Defines the width of the sidewalk associated to each way-point (if there are less values than way-points, the last value is used for the last remaining way-points).
```

File: projects/objects/road/protos/Roundabout.proto [hunk 3]
```diff
@@ -14,7 +14,7 @@ PROTO Roundabout [
   field        SFFloat              speedLimit               -1.0                                  # Optionally defines the speed limit. The recommended unit is meter per seconds.
   field        SFBool               bottom                   FALSE                                 # Defines whether the road bottom should be displayed (useful in case of bridge).
   field        SFBool               border                   TRUE                                  # Defines whether the roundabout should have sidewalks.
-  field        SFFloat              borderWidth              0.8                                   # Defines the width of the sidewalks.
+  field        SFFloat              borderWidth              0.8                                   # Defines the width of the sidewalk.s.
```
Comment (stefaniapedrazzi) [inline projects/objects/road/protos/Roundabout.proto]:
```suggestion
  field        SFFloat              borderWidth              0.8                                   # Defines the width of the sidewalks.
```

File: projects/objects/road/protos/StraightRoadSegment.proto [hunk 4]
```diff
@@ -19,8 +19,8 @@ PROTO StraightRoadSegment [
   field        SFFloat              speedLimit                -1.0                    # Optionally defines the speed limit. The recommended unit is meter per seconds.
   field        MFNode{RoadLine {}}  lines                     [ RoadLine { } ]        # Defines the property of each line separating two lanes.
   field        SFFloat              roadBorderHeight          0.15                    # Defines the height of the sidewalk.
-  field        SFFloat              startingRoadBorderWidth   0.8                     # Defines the width of the sidewalk at the start.
-  field        SFFloat              endingRoadBorderWidth     0.8                     # Defines the width of the sidewalk at the end.
+  field        SFFloat              startingRoadBorderWidth   0.8                     # Defines the width of the sidewalk. at the start.
```
Comment (stefaniapedrazzi) [inline projects/objects/road/protos/StraightRoadSegment.proto]: Why did you modify this line?
It seems that it was correct before:
```suggestion
  field        SFFloat              startingRoadBorderWidth   0.8                     # Defines the width of the sidewalk at the start.
```

File: projects/objects/road/protos/StraightRoadSegment.proto [hunk 5]
```diff
@@ -19,8 +19,8 @@ PROTO StraightRoadSegment [
...
+  field        SFFloat              endingRoadBorderWidth     0.8                     # Defines the width of the sidewalk. at the end.
```
Comment (stefaniapedrazzi) [inline projects/objects/road/protos/StraightRoadSegment.proto]: Why did you modify this line?
It seems that it was correct before:
```suggestion
  field        SFFloat              endingRoadBorderWidth     0.8                     # Defines the width of the sidewalk at the end.
```

File: projects/vehicles/protos/abstract/Car.proto [hunk 6]
```diff
@@ -45,14 +45,14 @@ PROTO Car [
...
+  field SFFloat    engineMaxPower                 50000                      # Defines the maximum power of the motor [W] used to compute the electric engine torque.
+  field SFFloat    engineMinRPM                   1000                       # Defines the minimum RPM of the motor (only used in the case of combustion engine).
   field SFFloat    engineMaxRPM                   4500
-  field SFVec3f    engineFunctionCoefficients     150 0.1 0  # used to compute the combustion engine torque
+  field SFVec3f    engineFunctionCoefficients     150 0.1 0                  # Defines the engine coefficients used to compute the combustion engine torque.
   field MFFloat    gearRatio                      [-12 10 6.5 4.5 3.5 2.75]
-  field SFFloat    hybridPowerSplitRatio          0.25 # only used in the case of 'power-split hybrid' engineType
-  field SFFloat    hybridPowerSplitRPM            3000 # only used in the case of 'power-split hybrid' engineType
+  field SFFloat    hybridPowerSplitRatio          0.25                       # Defines power split ration (the only used in the case of 'power-split hybrid' engineType).
```
Comment (stefaniapedrazzi) [inline projects/vehicles/protos/abstract/Car.proto]: typo
```suggestion
  field SFFloat    hybridPowerSplitRatio          0.25                       # Defines power split ratio (only used in the case of 'power-split hybrid' engineType).
```

File: projects/devices/multisense/protos/MultiSenseS21.proto [hunk 7]
```diff
@@ -11,7 +11,7 @@ PROTO MultiSenseS21 [
   field SFString   name                "MultiSense S21"
   field SFInt32    cameraWidth         1024
   field SFInt32    cameraHeight        544
-  field SFFloat    cameraFieldOfView   1.4  # = 80 [deg]
+  field SFFloat    cameraFieldOfView   1.4               # The default value is equal to 80deg.
```
Comment (stefaniapedrazzi) [inline projects/devices/multisense/protos/MultiSenseS21.proto]: is not better to add a space?
```suggestion
  field SFFloat    cameraFieldOfView   1.4               # The default value is equal to 80 deg.
```

File: projects/objects/road/protos/Road.proto (hunk 1 above)
Comment (DavidMansolino) [reply projects/objects/road/protos/Road.proto]: Well spotted, it is a search and replace issue (another comment was only `Defines the width of the sidewalk`, https://github.com/cyberbotics/webots/pull/1286/files#diff-09ea5430b92d786c1d8c174a878da623L21).
